repair_part_source_location/models/stock_move.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Camptocamp SA (https://www.camptocamp.com).
+# @author: Italo Lopes <italo.lopes@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+from odoo.addons.repair.models.stock_move import (
+    MAP_REPAIR_LINE_TYPE_TO_MOVE_LOCATIONS_FROM_REPAIR,
+)
+
+MAP_REPAIR_LINE_TYPE_TO_MOVE_LOCATIONS_FROM_REPAIR["add"] = {
```

Comment:
I have weird feelings about patching this.

It seems this variable is only used in the `stock.move._get_repair_locations` method, and that seems easy to override.

I think it'd be much cleaner to do that instead